pkg/controller/service/service_controller.go

```diff
@@ -0,0 +1,178 @@
+package service
+
+import (
+	"context"
+
+	cjfrapi "github.com/rh-jmc-team/container-jfr-operator/pkg/apis/rhjmc/v1alpha1"
```

Comment:
why `cjfrapi` instead of `v1alpha1` ?  By referencing the apis by explicitly by version, you can make sure your controller/reconciliation can upgrade versions smoothly (by changing to `v1beta1` and `v1` explicitly and adjusting to the change in struct types).  Leaving it as `cjfrapi` means, internally, you'll have to add a variety of conditional logic based on the api version you're referencing, which can get unwieldy.

Reply:
Sounds good. I have renamed the import to rhjmcv1alpha1, which is already used elsewhere.